static/src/javascripts-legacy/projects/common/modules/experiments/tests/acquisitions-epic-election-interactive-end.js

```diff
@@ -6,13 +6,16 @@ define([
     'lib/config',
     'raw-loader!common/views/acquisitions-epic-control.html',
     'raw-loader!common/views/acquisitions-epic-slice.html',
+    'common/modules/commercial/acquisitions-copy',
+
 ], function (
     contributionsUtilities,
     $,
     geolocation,
     template,
     config,
-    epicControlTemplate
+    epicControlTemplate,
+    acquisitionsCopy
```

Comment:
Won't `'raw-loader!common/views/acquisitions-epic-slice.html'` be imported as `acquisitionsCopy` ?